modules/op-event-pruner-configuration.adoc

```diff
@@ -0,0 +1,98 @@
+// This module is included in the following assemblies:
+// * install_config/customizing-configurations-in-the-tektonconfig-cr.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="event-pruner-configuration_{context}"]
+= Enabling the event-based pruner
+
+:FeatureName: The event-based pruner
+include::snippets/technology-preview.adoc[]
+
+You can use the event-based `tektonpruner` controller to automatically delete completed resources, such as `PipelineRuns` and `TaskRuns`, based on configurable policies. Unlike the default job-based pruner, the event-based pruner listens for resource events and prunes resources in near real time.
+
+You can use the event-based pruner alongside:
+
+* Resource annotations
+* {tekton-results}
+* {PAC}
+
+[NOTE]
+====
+The Operator does not currently restrict the use of these features with the event-based pruner. However, future releases might introduce validation if conflicts are observed.
+====
+
+[IMPORTANT]
+====
+You must disable the default pruner in the `TektonConfig` custom resource (CR) before you enable the event-based pruner. If both pruners are enabled, the deployment readiness status changes to `False` and output reads the following error message:
+
+[source,terminal]
+----
+Components not in ready state: Invalid Pruner Configuration!! Both pruners, tektonpruner(event based) and pruner(job based) cannot be enabled simultaneously. Please disable one of them.
+----
+====
+
+.Procedure
+
+. In your TektonConfig CR, disable the default pruner by setting the `spec.pruner.disabled` field to `true` and enable the event-based pruner by setting the `spec.tektonpruner.disabled` field to `false`.  
++
+.TektonConfig CR with the event-based pruner enabled
```

Comment:
As per the new DITA guidelines, I think we do not need to provide titles to the YAML code sample. You can simply remove ".TektonConfig CR with the event-based pruner enabled".